src/posix/platform/radio_spinel.hpp

```diff
@@ -439,6 +440,9 @@ class RadioSpinel
     otError PlatDiagProcess(const char *aString, char *aOutput, size_t aOutputMaxLen);
 #endif
 
+    // ABTIN: TODO:
+    void HandleSpinelFrame(const uint8_t *aBuffer, uint16_t aLength);
```

Comment:
what to do?

Reply:
Ah. Thanks. I missed that.  :)
Added this not to forget adding documentation for the method...(which looks like I still forgot)...